CONTRIBUTING.md

```diff
@@ -12,39 +12,45 @@
+  - Che dashboard: UI to manage workspaces, devfiles, etc.
+  - Che main container images: source code, dockerfiles to build our main docker images. Note that Che-theia related dockerfiles are located in che-theia repo.
+  - End2end test: source code of our integration tests.
 - [che-theia](https://github.com/eclipse/che-theia): Theia IDE integrated in Che.
 - [chectl](https://github.com/che-incubator/chectl): The CLI to install Che, create and start workspaces and devfiles
 - [che-plugin-registry](https://github.com/eclipse/che-plugin-registry): The default set of che plugins (vscode extension + containers) or editors that could be installed on any Che workspaces.
 - [che-devfile-registry](https://github.com/eclipse/che-devfile-registry): The default set of devfiles that would be made available on the Che dashboard stacks.
-- [redhat-developer/devfile](https://github.com/redhat-developer/devfile): Contribute to the devfile documentation: https://redhat-developer.github.io/devfile/
+- [redhat-developer/devfile](https://github.com/redhat-developer/devfile): Contribute to the devfile documentation: `https://redhat-developer.github.io/devfile/`
 - [che-plugin-broker](https://github.com/eclipse/che-plugin-broker): The workspace microservice that is in charge of analyzing, preparing and installing the workspace components defined in a Devfile.
 - [che-operator](https://github.com/eclipse/che-operator): Che operator to deploy, update and manage K8S/OpenShift resources of Che.
-- [che-docs](https://github.com/eclipse/che-docs): Eclipse Che documentation https://www.eclipse.org/che/docs/ source code.
-- [che-website](https://github.com/eclipse/che-website): https://eclipse.org/che website source code.
+- [che-docs](https://github.com/eclipse/che-docs): Eclipse Che documentation `https://www.eclipse.org/che/docs/` source code.
+- [che-website](https://github.com/eclipse/che-website): `https://eclipse.org/che` website source code.
 - [che-workspace-client](https://github.com/eclipse/che-workspace-client): JS library to interact with a che-server.
 - [che-machine-exec](https://github.com/eclipse/che-machine-exec): Interface to execute tasks and terminals on other containers within a workspace.
 
 ## Devfile to contribute
+
 We are trying to provide a devfile for each areas where you could contribute. Each devfile could be run on any Che instances to setup a *ready-to-code* developer environment. Beware that each of them may need a certain amount of memory.
 Devfile could be launched through a factory or [chectl](https://github.com/che-incubator/chectl) cli.
 
 ```bash
-$ chectl workspace:start -f devfiles/che-theia-all.devfile.yaml
+chectl workspace:start -f devfiles/che-theia-all.devfile.yaml
 ```
+
 or
+
 ```bash
-$ chectl workspace:start -f https://raw.githubusercontent.com/eclipse/che-theia/master/devfiles/che-theia-all.devfile.yaml
+chectl workspace:start -f https://raw.githubusercontent.com/eclipse/che-theia/master/devfiles/che-theia-all.devfile.yaml
```

Comment:
Why Did you remove `$` in a few places? For most users this character associated with command line interface.

Reply:
done